apps/mobile/src/state/chat/atoms/sendMessageToChatsInBatchActionAtom.ts

```diff
@@ -31,7 +32,7 @@ const sendMessageToChatsInBatchActionAtom = atom(
     set,
     {messageData, chats, isTerminationMessage}: Params
   ): TE.TaskEither<
-    ExtractLeftTE<ReturnType<typeof sendMessagesBatch>>,
+    ExtractErrorFromEffect<ReturnType<typeof sendMessagesBatch>>,
     boolean
   > => {
```

Comment:
Check if this function is still in use somewhere after rebase. If not delete this...